_sass/layouts/_nominating-candidates.scss

```diff
@@ -0,0 +1,8 @@
+@media print {
+    main {
+        max-width: 100%
+    }
+    .header, .site-footer, p, .social-links, caption {
```

Comment:
Here I think I wouldn't hide ALL captions, but just the first one, because it's legally mandatory for employees to know "which election/date, workplace etc.." but this can be refined later. Just adding a note here not to forgot